usaspending_api/common/helpers/download_csv_strategies.py

```diff
@@ -108,7 +116,7 @@ def download_to_csv(
             raise e
         finally:
             Path(temp_file_path).unlink()
-        return [destination_path], count
+        return CSVDownloadMetadata([destination_path], row_count)
```

Comment:
Not worth fixing as part of this work, but worth calling out so that we remember. This change to return a dataclass will mean that the current download functionality will need to take this new return into account as well.